docs/output.md

```diff
@@ -1,22 +1,21 @@
 # Output
 
 This document describes the output produced by the coproID pipeline.
+Results are found in the `results` directory (default, specified by `--outdir`).
 
-## Pipeline overview
+## MultiQC report
 
-The pipeline is built using [Nextflow](https://www.nextflow.io/)
-and processes data using the following steps:
+File `multiqc_report.html`
 
-
-## FastQC
+### FastQC section
 
 [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
 
 For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
 
 > **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
 
-### AdapterRemoval
+### AdapterRemoval section
 
 [AdapterRemoval](https://github.com/MikkelSchubert/adapterremoval) searches for and removes remnant adapter sequences from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases from the 3' end of reads following adapter removal. AdapterRemoval can analyze both single end and paired end data, and can be used to merge overlapping paired-ended reads into (longer) consensus sequences.
```

Comment:
This is more a description of the tool, doesn't really help with what you're looking at with the output (which is clearer in the FastQC actual outuput because they split up over so many plots with titles)

Reply:
Added more details with 53e6392